lms/templates/ccx/grading_policy.html

```diff
@@ -3,7 +3,7 @@
 <div id="warn-coach" class="wrapper-msg urgency-high warning">
   <div class="msg">
     <i class="msg-icon icon-warning-sign"></i>
-    <div class="msg-content">
+    <div class="msg-content" tabindex="0">
```

Comment:
@amir-qayyum-arbisoft or @pwilkins can you please respond to this above? Also can you please help Mark understand how to test this feature?